test/e2e/es_cluster_logging.go

```diff
@@ -240,6 +240,18 @@ func ClusterLevelLoggingWithElasticsearch(f *Framework) {
 	missing := 0
 	expected := nodeCount * countTo
 	for start := time.Now(); time.Since(start) < graceTime; time.Sleep(10 * time.Second) {
+
+		// Debugging code to report the status of the elasticsearch logging endpoints.
+		esPods, err := f.Client.Pods(api.NamespaceDefault).List(labels.Set{"k8s-app": "elasticsearch-logging"}.AsSelector(), fields.Everything())
+		if err != nil {
+			Logf("Failed to find any elasticsearch logging nodes: %v", err)
```

Comment:
nit: This is not a real failure.  Please reword to something like "No elasticsearch logging nodes found yet, retrying...".

We're definitely not consistent about this yet, but I plan to reword all messages implying failure that are not actually failures, to make failure diagnosis much easier.  This will be one less to fix :-)